react/common/constants/returnsRequest.ts

```diff
@@ -0,0 +1,68 @@
+import { defaultReturnReasonsMessages } from '../../store/utils/defaultReturnReasonsMessages'
+
+export function getReasonOptions(formatMessage) {
```

Comment:
Can you type `formatMessage`, please? This is fine now because I turned off a flag to allow `any` but I am planning to turn it on when we get rid of the old files.

Reply:
Done